addons/account/models/partner.py

```diff
@@ -434,6 +434,11 @@ def _get_company_currency(self):
 
     credit = fields.Monetary(compute='_credit_debit_get', search=_credit_search,
         string='Total Receivable', help="Total amount this customer owes you.")
+    credit_limit = fields.Monetary(string='Internal Credit Limit')
+    credit_limit_compute = fields.Monetary(
+        string='Credit Limit', company_dependent=False,
+        compute='_compute_credit_limit_compute', inverse='_inverse_credit_limit_compute')
```

Comment:
I am curious, why is there 2 differents fields?
Is it because you use this one to set the first ? via the `_inverse_credit_limit_compute` ?

Reply:
That's the tricky part that I was missing at first, when both were merged into one field.
It's because we need to store the value set on the partner, and we also want the field to be recomputed every time the view is loaded (in case the limit on the company was changed).
But since we depend on the context (i.e. company), the `depend_context` did not trigger the recompute of the field.
Credit to @william-andre for the solution :slightly_smiling_face: